frontend/app/views/fragments/tier/packagePromoSupporter.scala.html

```diff
@@ -6,6 +6,7 @@
 @import com.gu.i18n.CountryGroup
 @import com.gu.memsub.subsv2.MonthYearPlans
 @import com.gu.memsub.subsv2.CatalogPlan.PaidMember
+
```

Comment:
Try to avoid whitespace-only changes as it distracts from the substantial changes in the PR